malasakit-django/pcari/templates/base.html

```diff
@@ -36,6 +37,18 @@
         <h1 id="main-heading">{% block main-heading %}Malasakit{% endblock %}</h1>
       </div>
     </header>
+    <page-summary>
+      <div class="container">
+        <center>
```

Comment:
Aaaaaaaaaaa text-align is evil and broken. *sob*
This makes me sad but I'll do it.